### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by EduPonz**
This PR takes the work done in #503 and applies some requested changes

### CODE REVIEW

**Comment by EduPonz** [comment]
DCO fails due to difference between signer and committer. @hidmic should I signed the commits?

**Comment by hidmic** [comment]
@EduPonz if I'm not mistaken, the problem is the discrepancy between the previous author and his previous sign. You may sign them as well, but I won't solve the DCO check failures.

File: README.md
Comment (hidmic) [inline README.md:66]: @EduPonz nit^2: 
```suggestion
In that case, XML values, or Fast DDS default values in the absence of XML ones, are applied.
```

**Review by hidmic — APPROVED** [review]
LGTM pending DCO resolution

File: README.md
Comment (EduPonz) [reply README.md]: Done in 6209d7d

**Comment by EduPonz** [comment]
> @EduPonz if I'm not mistaken, the problem is the discrepancy between the previous author and his previous sign. You may sign them as well, but I won't solve the DCO check failures.

I think you're right. I've signed the changes just in case anyways. I don't what else can be done other than remove the original author's and make their changes as if I did them. What's your suggestion?

**Comment by EduPonz** [comment]
I stand corrected, signing them all myself did the trick!

**Comment by EduPonz** [comment]
Hi @hidmic ,

Do you want me to do something else here?

**Comment by clalancette** [comment]
I'm going to go ahead and merge this since hidmic is unavailable this week.  Also, since it is a change to the README.md any follow-ups should be easy.  Thanks for the contribution!

**Comment by hidmic** [comment]
Thanks for taking care @clalancette

**Pull request merged**
